rules-reviewed/eap7/eap6/tests/data/wsdl-mapping.xml

```diff
@@ -0,0 +1 @@
+<java-wsdl-mapping/>
```

Comment:
If you reintroduce the namespaces per my comment above, here's a testing file for this (copied directly from `j2ee_jaxrpc_mapping_1_1.xsd`):

```xml
<java-wsdl-mapping xmlns="http://java.sun.com/xml/ns/j2ee"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee
        http://www.ibm.com/webservices/xsd/j2ee_jaxrpc_mapping_1_1.xsd"
    version="1.1">
</java-wsdl-mapping>
```